neutron_plugin/port.py

```diff
@@ -73,7 +75,7 @@ def create(neutron_client, **kwargs):
         except Exception:
             delete_runtime_properties(ctx, RUNTIME_PROPERTIES_KEYS)
             raise
-
+    ctx.logger.info('[all connected nodes]INFO: {0}'.format(str(ctx.capabilities.get_all().values())))
```

Comment:
nonstandard error message, I think you should drop the "INFO:" inside the message body